packages/lodestar/src/chain/chain.ts

```diff
@@ -86,16 +86,15 @@ export class BeaconChain extends (EventEmitter as { new(): ChainEventEmitter })
   }
 
   public async getHeadState(): Promise<BeaconState|null> {
-    return this.db.state.get(this.forkChoice.headStateRoot());
+    return this.db.state.get(this.forkChoice.headStateRoot().valueOf() as Uint8Array);
```

Comment:
shouldn't need these `valueOf`s.

We only need `valueOf` to convert `TreeBacked<Vector<number>>` to `Uint8Array`